spec/presenters/tree_builder_alert_profile_obj_spec.rb

```diff
@@ -0,0 +1,126 @@
+describe TreeBuilderAlertProfileObj do
+  before do
+    role = MiqUserRole.find_by(:name => "EvmRole-operator")
+    group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
+    login_as FactoryGirl.create(:user, :userid => 'tags_wilma', :miq_groups => [group])
+  end
```

Comment:
I see you already told me to do this and to change it back, so you did mistake before, or you had some purpose before you forgot that?